frontend/src/components/Home.js

```diff
@@ -34,11 +34,48 @@ const Home = () => {
     }
   };
 
+
+  fetch("/api/user", {
```

Comment:
Why is this function still here? It's been moved to a util and is called on mount?

Reply:
Oh, okay